test_projects/android/ops.sh

```diff
@@ -23,4 +23,8 @@ function cucumber_sample_app() {
   $DIR/../../flank-scripts/bash/flankScripts assemble android app --copy --generate --artifacts=buildCucumberSampleApp
 }
 
+function many_tests_apk() {
```

Comment:
Do we have any good reasons for supporting shell scripts, now? Wasn't a `flank-scripts` a replacement for the shell? 

This is not a change request, I am just thinking about adding an issue for removing any `ops.sh`

Reply:
`ops.sh` just run `flankScripts` with the proper command it was added for backward compatibility, but sure we could remove it

Reply:
+1 to not having shell scripts if possible